registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -138,6 +143,13 @@
 
 	@Value("${mosip.regproc.data.share.internal.domain.name}")
 	private String internalDomainName;
+	
+	
+	@Value("#{${mosip.regproc.data.share.modalities.segments}}")
+	private Map<String, List<String>> biometricModalitySegmentsMap;
+	
+	@Value("#{${mosip.regproc.data.share.segments.exception}}")
+	private Map<String, String> exceptionModalityMap;

```

Comment:
Rename this config to mosip.regproc.data.share.biometric-exception-segments-mapping. Rename the variable also to exceptionSegmentsMap

Reply:
Renamed variable